tests/integration/test_system_containers_images.sh

```diff
@@ -78,7 +81,9 @@ assert_not_matches '<none>' ${WORK_DIR}/images.out
 
 
 # 3. deleting/pruning local images
-${ATOMIC} --assumeyes images delete -f --storage ostree busybox
+${ATOMIC} --assumeyes images delete -f --storage ostree busybox > ${WORK_DIR}/image.delete.out
+assert_matches "NO" ${WORK_DIR}/image.delete.out
+assert_matches "USED" ${WORK_DIR}/image.delete.out
```

Comment:
Good point, my intention was to put it there so people can see the USED tag was there without downloading the repo and see it :p